modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt

```diff
@@ -121,18 +121,20 @@ class PodcastSyncProcess(
             }
         }
         val syncUpNextObservable = downloadObservable
-            .andThen(syncSettings(lastSyncTime))
-            .andThen(syncCloudFiles())
-            .andThen(firstSyncChanges())
+            .andThen(rxCompletable { syncSettings(lastSyncTime) })
+            .andThen(rxCompletable { syncCloudFiles() })
+            .andThen(rxCompletable { firstSyncChanges() })
```

Comment:
Instead of having the method to convert to Rx in the method I have moved them here ready to be removed in a future change.